### ISSUE TITLE
Fix Zenodo bug caused by changes.

### ISSUE THREAD

**Issue by mr-devs** [osome-iu/top-FIBers#20]
From the master script log for the November report.

```
2023-11-01_11:36:28-update_zenodo.py-INFO - --------------------------------------------------
2023-11-01_11:36:28-update_zenodo.py-INFO - Begin script: /home/data/apps/topfibers/repo/scripts/update_zenodo.py
2023-11-01_11:36:31-update_zenodo.py-INFO - Retrieved all deposition info
2023-11-01_11:36:31-update_zenodo.py-INFO - Failed to create draft. Status code: 404
2023-11-01_11:36:31-update_zenodo.py-INFO - {'status': 404, 'message': 'The persistent identifier does not exist.'}
```

@Nick-the-freak mentioned that Zenodo is going through some changes which broke the script, is investigating potential solutions, and will make a fix.

**Comment by mr-devs** [comment]
Hey @Nick-the-freak — any luck looking into this? Thanks!

**Comment by nick-c-liu** [comment]
this was fixed in commit [f6da977](https://github.com/osome-iu/top-FIBers/commit/2af65a9285c6e7187bc8fca2c20dd38bc440dce2), error originated from server updates on Zenodo end
